tests/default.nix

```diff
@@ -27,8 +27,7 @@ in {
   # the machine config doesn't have the overlay and the test is always running
   # the upstream version of docker, including dependencies.
   # When in doubt, it's better to write our own test or copy&paste from nixpkgs.
-  inherit (pkgs.nixosTests)
-    matomo;
+  # inherit (pkgs.nixosTests)
```

Comment:
Do we plan to adopt this again in the future? Otherwise: delete.

Reply:
we will probably need this later, makes sense to keep it.